site/docs/4.1/assets/scss/_anchor.scss

```diff
@@ -1,10 +1,15 @@
-.anchorjs-link {
+.bd-anchor {
+  padding-left: .25rem;
   font-weight: 400;
-  color: rgba($link-color, .5);
-  @include transition(color .16s linear);
+  color: $link-color;
+  opacity: 0;
```

Comment:
Can someone verify if this needs `visibility: hidden` too?

Reply:
It doesn't. You can't transition visibility: hidden, but you can transition the opacity.